### ISSUE TITLE
Bug: Unable to import using ESM

### ISSUE THREAD

**Issue by miichom** [AwesomeHamster/lodestone-news#26]
### What happened?

```
(node:250600) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
```
```
SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
```

### What do you expect?

A simple (but temporay) fix for me was to change `/dist/esm/index.js` to `/dist/esm/index.mjs` and update the path in `package.json`

```diff
- "module": "dist/esm/index.js",
+ "module": "dist/esm/index.mjs",
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MaikoTan** [comment]
There are few questions:

- Are you using `require()` or `import` to import the package?
- Are you using any code bundler for your project?
- Do you using `type: module` or `type: commonjs` in your root `package.json`?

**Comment by miichom** [comment]
> * Are you using `require()` or `import` to import the package?

In the affect project I use ESM `import`
> * Are you using any code bundler for your project?

I code in typescript using the `node` module resolution, it hasn't proven difficult.
> * Do you using `type: module` or `type: commonjs` in your root `package.json`?

I use `type: module`, this is a bug with ESM and not CJS - as I have tested it using CJS in both the affect and outside of it.

**Comment by MaikoTan** [comment]
Hello @c0reme , I have published the version [`0.2.0`](https://www.npmjs.com/package/lodestone-news) with the fixing of importing error. Is this version works for you then?

**Comment by miichom** [comment]
Yup that version works perfectly, thank you 👍
